Create 18-sponsored-memberships.md (#1393)
carpentries/carpentries.org: _posts/2022/05/18-sponsored-memberships.md

---
layout: page
authors: ["Membership Team"]
teaser: "We invite institutions serving underrepresented groups in Central and South America and in Southern Africa to apply for sponsored membership."
title: "Applications for sponsored Carpentries Memberships open now!"
date: 2022-05-18
time: "09:00:00"
tags: ["Carpentries Membership"]
---

During our Executive Director, Dr. Kari L. Jordan's, [New Year address](https://carpentries.org/blog/2022/01/executive-director-new-years-message/) we annouced funding 

> from the Chan Zuckerberg Initiative to provide **3 silver Carpentries memberships** to institutions serving underrepresented groups in Central and South 

Reviewer: ```suggestion
from the Chan Zuckerberg Initiative to provide **three (3) Silver Carpentries memberships** to institutions serving underrepresented groups in Central and South 
```

> America and in Southern Africa for **2 years**. A silver membership includes **2 Centrally-Organised workshops** and **5 instructors to be trained** in online events. 

Reviewer: ```suggestion
America and in Southern Africa for **2 years**. A silver membership includes **2 Centrally-Organised workshops** and **5 seats in online Instructor Training**.
```

> America and in Southern Africa for **2 years**. A silver membership includes **2 Centrally-Organised workshops** and **5 instructors to be trained** in online events. 

Reviewer: ```suggestion
America and in Southern Africa for **two (2) years**. A Silver membership includes **two (2) Centrally-Organised workshops** and **five (5) seats in online Instructor Training**. 
```
